.kokoro/publish.sh

```diff
@@ -28,4 +28,31 @@ NPM_TOKEN=$(cat $KOKORO_GFILE_DIR/secret_manager/npm_publish_token)
 echo "//wombat-dressing-room.appspot.com/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 
 npm install
+
+# Publish the default package
+npm publish --access=public --registry=https://wombat-dressing-room.appspot.com
+
+# Uglify /build to publish a smaller package
+for d in $(find ./build/ -maxdepth 8 -type d)
+do
+  pushd $d
+    for f in *.js; do
+      [ -f "$f" ] || break
+
+      if [ -f "$f.map" ]; then
+        # Keep original .ts source mappings
+        uglifyjs --source-map "content='$f.map', url='$f.map'" "$f" --output "$f"
+      else
+        uglifyjs "$f" --output "$f"
+      fi
+    done
+  popd
+done
+
+# Change and publish under package name `@google-cloud/logging-min`
+sed -i -e 's/"name": "@google-cloud\/logging"/"name": "@google-cloud\/logging-min"/' package.json
```

Comment:
This seems a little hacky to me, I was picturing we'd perhaps just make a new repository for this use case.

Is this the eventual plan?

Reply:
Yes this is the eventual plan. Doc shared for your review in our chat.